<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Frequency Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStop">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHide">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pbStart</tabstop>
  <tabstop>pbStop</tabstop>
  <tabstop>pbHide</tabstop>
  <tabstop>pbQuit</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
